lib/src/generated_serialization_rules.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/**
+ * This is a template file that will be copied to the projects that use
+ * transformer-based serialization and will be filled with the generated
+ * serialization rules.
+ */
+library generated_serialization_rules;
+
+// ====== Imports generated by the transformer will be appended below =======
+// ======== Do not remove the {{GENERATED_IMPORTS_MARKER}} comment ==========
+
+import 'package:serialization/serialization.dart' as s show Serialization;
+import 'package:serialization/serialization.dart' hide Serialization;
+export 'package:serialization/serialization.dart';
+//{{GENERATED_IMPORTS_MARKER}}
```

Comment:
I think it's much better to have this "initial" code as a real file vs a String in code. This way you will be able to take advantage of some IDEs features like refactoring and errors display if you do non-backward compatible changes in.